docs/menu.yml

```diff
@@ -43,6 +43,10 @@ catalog:
             path: "/en/concepts-and-designs/backend-overview"
           - name: "Observability Analysis Language"
             path: "/en/concepts-and-designs/oal"
+          - name: "Meter Analysis Language"
```

Comment:
Let's also add this file to https://github.com/apache/skywalking/blob/1e4a8ee19b1c5d43f93b2c0dea5c6bd92a9e4f83/.github/actions/skip/action.yml#L27

Reply:
Added.